ecs_model_deployer/.gitignore

```diff
@@ -0,0 +1,2 @@
+dist/
```

Comment:
Is a new .gitignore needed for the project? Is the toplevel one not sufficient?

Reply:
If we wanted to get rid of this, we'd need to do `ecs_model_deployer/dist`, or `**/dist`. The former would break if we renamed the directory, the latter feels a little dangerous for a monorepo, which is why I added the nested `.gitignore`. If you prefer one of the solutions from here though that would also totally work so just let me know and I'll update it.

Reply:
all good- i haven't seen a nested gitignore like that before 😅